readme.md

```diff
@@ -14,7 +14,7 @@ If you like Nette, **[please make a donation now](https://nette.org/donate)**. T
 Requirements
 ------------
 
-PHP 5.6 or higher.
+PHP 7.1 or higher.
```

Comment:
This is readme.md for all branches, not only for master branch. So even better is to provide full info about requirement, something like 

`Nette Web-Project for Nette 2.4 requires PHP 5.6. The dev-master version requires PHP 7.1.`

Reply:
fixed